.gitignore

```diff
@@ -5,3 +5,8 @@ cover.out
 test/sender/sender
 test/receiver/receiver
 
+# Fuzzing artefacts.
+corpus
+crashers
+suppressions
+*.zip
```

Comment:
IMO, this is a bit too generic of an ignore.

Reply:
I've put everything fuzzing–related into a separate directory under test/. Won't clash with any zip files you may to commit to the repo in the future 😀